packages/rainbowkit/src/locales/en_US.json

```diff
@@ -0,0 +1,640 @@
+{
+  "intro": {
+    "title": "What is a Wallet?",
+    "digital_asset": {
+      "title": "A Home for your Digital Assets",
+      "description": "Wallets are used to send, receive, store, and display digital assets like Ethereum and NFTs."
+    },
+    "login": {
+      "title": "A New Way to Log In",
+      "description": "Instead of creating new accounts and passwords on every website, just connect your wallet."
+    },
+    "ethereum": {
+      "title": "New to Ethereum wallets?"
+    },
+    "install": {
+      "label": "INSTALL"
+    },
+    "group_name": {
+      "recommended": "Recommended",
+      "other": "Other",
+      "popular": "Popular",
+      "more": "More",
+      "others": "Others"
+    },
+    "wallet": {
```

Comment:
Let's have a more clear key hierarchy for maintenance. The `intro` key should only cover this UI element. The "Get a wallet" flow should be a separate top-level key. Same for "Connect a Wallet" sidebar, and any drill-down flow. WalletConnectors will have their own top-level key, but the UI flows should otherwise be reflected like other UI flows.
<img width="434" alt="Screenshot 2023-10-02 at 4 30 59 PM" src="https://github.com/rainbow-me/rainbowkit/assets/4412473/46a4e77e-e053-4c70-a4b4-a5799edc659a">